contracts/optimistic-ethereum/iOVM/bridge/ERC20/iOVM_L2ERC20Gateway.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
```

Comment:
any reason to not allow v0.8?

Reply:
i think their forked version of `solc` only supports up to 0.7.4 atm

Reply:
no reason not to remove from the interface though, thanks justin!